### PR TITLE
Feature/Remove empty constructors

### PR DESCRIPTION
**Pull request by dtslvr**

### CODE REVIEW

**Comment by dtslvr** [comment]
cc @dw-0

File: apps/client/src/app/components/dialog-footer/dialog-footer.component.ts [hunk 1]
```diff
@@ -19,8 +19,6 @@ export class DialogFooterComponent implements OnInit {
 
   @Output() closeButtonClicked = new EventEmitter<void>();
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/components/dialog-footer/dialog-footer.component.ts:22]: This empty OnInit could be removed as well

File: apps/client/src/app/components/dialog-header/dialog-header.component.ts [hunk 2]
```diff
@@ -21,8 +21,6 @@ export class DialogHeaderComponent implements OnInit {
 
   @Output() closeButtonClicked = new EventEmitter<void>();
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/components/dialog-header/dialog-header.component.ts:24]: This empty OnInit could be removed as well

File: apps/client/src/app/components/fear-and-greed-index/fear-and-greed-index.component.ts [hunk 3]
```diff
@@ -21,8 +21,6 @@ export class FearAndGreedIndexComponent implements OnChanges, OnInit {
   public fearAndGreedIndexEmoji: string;
   public fearAndGreedIndexText: string;
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/components/fear-and-greed-index/fear-and-greed-index.component.ts:24]: This empty OnInit could be removed as well

File: apps/client/src/app/components/portfolio-summary/portfolio-summary.component.ts [hunk 4]
```diff
@@ -35,8 +35,6 @@ export class PortfolioSummaryComponent implements OnChanges, OnInit {
   );
   public timeInMarket: string;
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/components/portfolio-summary/portfolio-summary.component.ts:38]: This empty OnInit could be removed as well

File: apps/client/src/app/components/toggle/toggle.component.ts [hunk 5]
```diff
@@ -34,8 +34,6 @@ export class ToggleComponent implements OnChanges, OnInit {
 
   public optionFormControl = new FormControl<string>(undefined);
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/components/toggle/toggle.component.ts:37]: This empty OnInit could be removed as well

File: apps/client/src/app/pages/faq/self-hosting/self-hosting-page.component.ts [hunk 6]
```diff
@@ -10,8 +10,6 @@ import { Subject } from 'rxjs';
 export class SelfHostingPageComponent implements OnDestroy {
   private unsubscribeSubject = new Subject<void>();
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/pages/faq/self-hosting/self-hosting-page.component.ts:13]: This empty OnInit could be removed as well

File: apps/client/src/app/pages/i18n/i18n-page.component.ts [hunk 7]
```diff
@@ -11,8 +11,6 @@ import { Subject } from 'rxjs';
 export class GfI18nPageComponent implements OnInit {
   private unsubscribeSubject = new Subject<void>();
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/pages/i18n/i18n-page.component.ts:14]: This empty OnInit could be removed as well

File: apps/client/src/app/pages/markets/markets-page.component.ts [hunk 8]
```diff
@@ -10,8 +10,6 @@ import { Subject } from 'rxjs';
 export class MarketsPageComponent implements OnDestroy, OnInit {
   private unsubscribeSubject = new Subject<void>();
 
-  public constructor() {}
-
   public ngOnInit() {}
```
Comment (dw-0) [inline apps/client/src/app/pages/markets/markets-page.component.ts:13]: This empty OnInit could be removed as well

**Review by dw-0 — COMMENTED** [review]
Overall looks good. I added some comments on spots where empty OnInits could be removed as well.

**Comment by dtslvr** [comment]
> Overall looks good. I added some comments on spots where empty OnInits could be removed as well.

There is a separate issue for that 🙂
https://github.com/ghostfolio/ghostfolio/issues/3959

**Review by dw-0 — APPROVED** [review]

**Pull request merged**